src/base/Module.zig

```diff
@@ -56,11 +57,13 @@ pub const Store = struct {
     }
 
     pub fn deinit(self: *Store) void {
-        for (self.modules.items.items(.ident_store)) |store| {
-            store.deinit();
+        const modules = self.modules.items;
+        for (0..self.modules.len()) |index| {
+            var module = modules.get(index);
+            module.exposed_idents.deinit();
         }
-
         self.modules.deinit();
+        self.ident_store.deinit();
```

Comment:
If we end up making `ident_store` not owned by this struct, we should remember to not `deinit` it here